### PR TITLE
removing envgen~ alias

### PR DESCRIPTION
**Pull request by porres**
I'm doing this cause it is pointless in a separately compiled binary setup

By the way, I don't think anybody knows about this, but envgen has an alias: envgen~

The fact that it has two names that are so similar is weird and could be thought of some sort of bug. And the thing is also that I don't believe anybody knows about it p I only lerned about it by checking the source code. And the fact is that you can't actually load the object as envgen~, unless you load it as [envgen] first - so, it's quite pointless, because it is an alias you can't actually use as you always need to load the object as [envgen] anyway, and the reason why I believe no one ever used this object as [envgen~].

If the case was that of a single binary pack, then it'd be different, as when you loaded the library you'd have access to all aliases and then load this as [envgen~] if you wished so, but that wasn't the case in Pd Extended and it is not the case now in Vanilla via deken... so... best thing is to just remove this from the code.

### CODE REVIEW

**Comment by reduzent** [comment]
I think you're reasoning makes sense and I merge your request. Thank you. I don't care that much about it to go through the trouble to make another release for that, though.

**Comment by porres** [comment]
> I don't care that much about it to go through the trouble to make another release for that, though.

What would make it worth it? A bug fix? There are a couple of other things I could do for [envgen], specially fix the documentation, which is misleading (remember I had an issue with sustain point)?

**Comment by reduzent** [comment]
Let me rephrase: It's not high on my priority list, so I rather won't create another release soon. However, if you feel strong about it and even fix stuff that needs fixing, I'll be supportive as much as I can. 

Let me also say: I don't want to "occupy" the maintenance of this library. I took it because I had a personal need for it. If you think it would be better in your hands, don't hesitate to maintain your fork and make releases out of it. The most important thing is that we know of each other what we're doing, so only one person pushes a certain release to Deken. Otherwise it would become confusing for the end user. You definitely should not think that I'm more entitled to make any decision about this library than you.

**Comment by porres** [comment]
> Let me also say: I don't want to "occupy" the maintenance of this library. I took it because I had a personal need for it. If you think it would be better in your hands, don't hesitate to maintain your fork and make releases out of it. The most important thing is that we know of each other what we're doing, so only one person pushes a certain release to Deken. Otherwise it would become confusing for the end user. 

Ok, I understand, and I'm really concerned about not making this more confusing. And I already think it is confusing for the end user as it is, with the other issue I opened at https://github.com/reduzent/pd-ggee/issues/1
